src/pages/css/colors/index.md

```diff
@@ -3,6 +3,12 @@ title: Colors
 ---
 ## Colors
 
+CSS Colors is a CSS module that deals with colors, color types, color blending, and opacity. Not all CSS properties that take a <color> as a value are part of this module, but they do depend upon it.
+Colors are specified using predefined color names, or RGB, HEX, HSL, RGBA, HSLA values.
+
+HTML supports 140 standard [color names](https://www.w3schools.com/colors/colors_names.asp).
+
+
 This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/colors/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```

Comment:
Hello, You need to replace the stub article with your new article and it would be great if you can add more details on this topic (optional)

Please read this style guide: https://github.com/freeCodeCamp/guides#article-style-guide

Please let me know if you have any questions.